app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/ExternalInsurerFragment.kt

```diff
@@ -39,6 +40,9 @@ class ExternalInsurerFragment : Fragment(R.layout.previous_or_external_insurer_f
     private val askForPriceActivityResultLauncher =
         registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result: ActivityResult ->
             if (result.resultCode == RESULT_CONTINUE) {
+                result.data?.getStringExtra(REFERENCE_RESULT)?.let {
+                    embarkViewModel.putInStore("dataCollectionId", it)
```

Comment:
iOS has hard coded this as well. I think we need to change something in embark. I'll talk to Sam.